src/view/output-view.js

```diff
@@ -7,62 +7,68 @@ const ACCESS_CONTROL_EXCEPTION = "java.security.AccessControlException";
 const SECURITY_MESSAGE = "Access control exception due to security reasons in web playground";
 const UNHANDLED_JS_EXCEPTION = "Unhandled JavaScript exception";
 const NO_TEST_FOUND = "No tests methods are found";
-const ANGLE_BRACKETS_LEFT_HTML = "&lt;";
-const ANGLE_BRACKETS_RIGHT_HTML = "&gt;";
 
 const TEST_STATUS = {
   FAIL : { value: "FAIL", text: "Fail" },
   ERROR: { value: "ERROR", text: "Error" },
   PASSED : { value: "OK", text: "Passed" }
 };
 
-const BUG_FLAG = `${ANGLE_BRACKETS_LEFT_HTML}errStream${ANGLE_BRACKETS_RIGHT_HTML}BUG${ANGLE_BRACKETS_LEFT_HTML}/errStream${ANGLE_BRACKETS_RIGHT_HTML}`;
-const BUG_REPORT_MESSAGE = `${ANGLE_BRACKETS_LEFT_HTML}errStream${ANGLE_BRACKETS_RIGHT_HTML}Hey! It seems you just found a bug! \uD83D\uDC1E\n` +
-  `Please click <a href=http://kotl.in/issue target=_blank>here<a> to submit it ` +
+const BUG_FLAG = 'BUG'
+const BUG_REPORT_MESSAGE = 'Hey! It seems you just found a bug! \uD83D\uDC1E\n' +
+  `Please go here -> http://kotl.in/issue to submit it ` +
```

Comment:
What is happened with link? :(

Reply:
Merging DOM nodes turns into text all inner elements so the link will not be clickable.
I thought that this case with `BUG_FLAG` is too rare and not really important to add specific behaviour for it. 
Is it possible to merge nodes without destroying children?